Library/Homebrew/cask/dsl/conflicts_with.rb

```diff
@@ -25,6 +25,12 @@ def initialize(pairs = {})
           instance_variable_set("@#{key}", instance_variable_get("@#{key}").merge([*value]))
         end
       end
+
+      def to_a
```

Comment:
Also this has to be renamed to `to_h`.

Reply:
Will have to update `cask.rb` in that case - it uses `to_a` or atleast expects it to be converted to an array instead of a hash.